src/main/scala/com/lightbend/paradox/projectinfo/model.scala

```diff
@@ -35,21 +35,27 @@ case class SbtValues(artifact: String,
 
 trait ReadinessLevel { def name: String }
 object ReadinessLevel {
+  private val glossary1 =
+    "<a href=\"https://developer.lightbend.com/docs/reactive-platform/2.0/support-terminology/index.html"
+  private val glossary2 = "\" target=\"_blank\" rel=\"noopener noreferrer\">"
+
   case object Supported extends ReadinessLevel {
     val name =
-      "<a href=\"https://www.lightbend.com/subscription\" target=\"_blank\">Lightbend Subscription</a> provides support"
+      s"""${glossary1}#supported${glossary2}Supported</a>, <a href="https://www.lightbend.com/subscription" target="_blank">Lightbend Subscription</a> provides support"""
```

Comment:
Maybe it would be a little easier to read and less error-prone to have a helper function like:

```scala
def glossary(anchor:String, label:String) =
  s"""<a href="https://developer.lightbend.com/docs/reactive-platform/2.0/support-terminology/index.html#${anchor}" target="_blank" rel="noopener noreferrer">${label}</a>"""
```